### PR TITLE
MAINT: simplify testing.assert_*, delegate to torch.assert_close

### PR DESCRIPTION
**Pull request by ev-br**
While at it, remove a bunch of dead code in testing/

### CODE REVIEW

File: torch_np/testing/utils.py [hunk 1]
```diff
@@ -207,924 +124,30 @@ def assert_equal(actual, desired, err_msg="", verbose=True):
...
-              instead of this function for more consistent floating point
-              comparisons.
+    from torch_np._normalizations import normalize_array_like
 
-    Given two numbers, check that they are approximately equal.
-    Approximately equal is defined as the number of significant digits
-    that agree.
+    actual = normalize_array_like(actual)
+    desired = normalize_array_like(desired)
+    return (actual == desired).all()
```
Comment (lezcano) [inline torch_np/testing/utils.py:132]: Is it expected that `assert_*` functions don't actually assert anything but just return a bool? That seems a bit weird to me.

Comment (lezcano) [reply torch_np/testing/utils.py:132]: Also, is it alright to return false for nans?

**Review by lezcano — COMMENTED** [review]
What's the reason for this big refactor? I'm a bit scared that we may introduce some bug in the testing facilities, which wouldn't be great.

**Comment by ev-br** [comment]
Per offline discussion with @lezcano , let's shelve this PR.

**Pull request closed without merging**